src/modules/dashboard/components/Extensions/ExtensionSetup.tsx

```diff
@@ -36,6 +39,13 @@ const MSG = defineMessages({
     id: 'dashboard.Extensions.ExtensionSetup.setPermissions',
     defaultMessage: 'Set permissions',
   },
+  complementaryLabel: {
+    id: 'dashboard.Extensions.ExtensionSetup.complementaryLabel',
+    defaultMessage: `{isPeriod, select,
+      true {hours}
+      false {%}
+    }`,
```

Comment:
You should consider adding another case for `other` which is a catch-all in case the value is neither true nor false _(maybe undefined)_

Reply:
Sounds good to me!